ledger/acctupdates.go

```diff
@@ -1071,6 +1071,23 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 	}
 }
 
+func (au *accountUpdates) lookupAccountDeltas(rnd basics.Round) (data ledgercore.AccountDeltas, err error) {
+	au.accountsMu.RLock()
+	defer au.accountsMu.RUnlock()
+	var offset uint64
+	currentDeltaLen := uint64(len(au.deltas))
+	offset, err = au.roundOffset(rnd)
+	if err != nil {
+		return
+	}
+	if currentDeltaLen < offset+1 {
+		err = fmt.Errorf("round %d not in deltas: dbRound %d, deltas %d, offset %d", rnd, au.cachedDBRound, len(au.deltas), offset)
+		return
+	}
+	data = au.deltas[offset]
+	return
+}
```

Comment:
nits: 
* I'm not sure the `currentDeltaLen` variable helps with readability.
* I'm not sure the named return values ever help with readability.
* I would consider `currentDeltaLen <= offset` over `currentDeltaLen < offset+1` to avoid multiple operators in the expression.